api/app/db/crud/observations.py

```diff
@@ -44,13 +44,13 @@ def get_actuals_left_outer_join_with_predictions(
                         ModelRunPrediction.station_code == station_code))\
```

Comment:
I think this check is unnecessary and can also be removed.
```
ModelRunPrediction.station_code == station_code
```

Reply:
Interesting if I remove that line from the SQL the results balloon up to over 1 million....not sure why yet

Reply:
Turns out that line is needed, otherwise we end up with data from all stations coming from the `model_run_predictions` table

Reply:
Thanks for checking!

Reply:
I suppose you probably meant we can just add it as a filter, rather than having it in the join. You're definitely right about that, would be a more logical way to do it